Jellyfin.Networking/Extensions/NetworkExtensions.cs

```diff
@@ -337,14 +335,9 @@ public static bool TryParseHost(ReadOnlySpan<char> host, [NotNullWhen(true)] out
     public static IPAddress GetBroadcastAddress(IPNetwork network)
     {
         var addressBytes = network.Prefix.GetAddressBytes();
-        if (BitConverter.IsLittleEndian)
-        {
-            addressBytes = addressBytes.Reverse().ToArray();
-        }
```

Comment:
This was removed because the byte order returned by `GetAddressBytes()` is always big endian:
https://stackoverflow.com/questions/1407938/ipaddress-getaddressbytes-method-what-byte-order